resources/views/dashboard/controllers/profile.blade.php

```diff
@@ -116,7 +116,8 @@
         <div class="col-sm-4">
             <h4>My Information</h4>
             <br>
-            <p><b>CID:</b> {{ Auth::id() }}</p>
+	    <p><b>CID:</b> {{ Auth::id() }}</p>
+	    <p><b>Name Privacy:</b> {{ Auth::user()->name_privacy == 1 ? 'Enabled' : 'Disabled' }} <a class="info-tooltip" href="https://www.vatusa.net/my/profile" target="_blank" data-toggle="tooltip" title="Click here to update"><i class="fas fa-info-circle"></i></a></p>
```

Comment:
@quackrr Please move the Name Privacy field just below Email and just before TS3 UID. My mistake for not making this more clear in the PR. When you've completed this, please tag me in Discord and I will review one final time. Thanks!